articles/api-auth/why-use-access-tokens-to-secure-apis.md

```diff
@@ -10,69 +10,39 @@ useCase:
-
-The portion of the login process where you "prove" your identity is implemented using OpenID Connect, while the part of the login process where you authorize the to-do application to modify your Google Calendar by adding entries is implemented using OAuth 2.0. 
-
-## The role of tokens
-
-You may have noticed that we've used the phrase **without sharing your credentials** several times in the paragraph above. How does this work?
-
-Essentially, the two protocols operate by sharing **tokens**.
-
-OpenID Connect issues an identity token, known as an ID Token, while OAuth 2.0 issues an Access Token.
-
-## How to use tokens
-
-The **ID Token** is a <dfn data-key="json-web-token">JSON Web Token (JWT)</dfn>, and it is meant for the application only. For example, in our calendar example above, Google sends an ID Token to the to-do app that tells the app who you are. The app then parses [the token's contents](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) and uses this information (including details like your name and your profile picture) to customize your user experience.
+The **ID Token** is a <dfn data-key="json-web-token">JSON Web Token (JWT)</dfn>, and it is meant for the application only. For example, in the example of a a to-do app that uses Google to log users in and to integrate their calendars with the app, Google sends an ID Token to the to-do app that tells the app who the user is and various pieces of information about them. The app then parses [the token's contents](https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims) and uses this information (including details like your name and your profile picture) to customize your user experience.
 
 ::: warning
 Be sure to [validate an ID Token](/tokens/guides/id-token/validate-id-token) before using the information it contains! You can use a [library](https://jwt.io/#libraries-io) to help with this task.
 :::
 
-The **Access Token** (which isn't necessarily a JWT), is meant for use by an API.
+## Access Tokens
+
+On the other hand, the **Access Token** (which isn't necessarily a JWT), is meant for use by an API.
 
 The Access Token's purpose is to inform the API that the bearer of the token has been authorized to access the API and perform a predetermined set of actions (which is specified by the <dfn data-key="scope">**scopes**</dfn> granted).
 
-In the Google/to-do app example above, recall that Google sent an Access Token to the to-do app after you logged in and provided consent for your to-do app to read/write to your Google Calendar.
+In the Google/to-do app example above, Google would have sent an Access Token to the to-do app after the user logged in and provided consent for the to-do app to read/write to your Google Calendar.
 
 Whenever the to-do app wants to write to your Google Calendar, it will send a request to the Google Calendar API, making sure to include the Access Token in the HTTP **Authorization** header.
```

Comment:
```suggestion
Whenever the to-do app wants to write to Google Calendar, it will send a request to the Google Calendar API, making sure to include the Access Token in the HTTP **Authorization** header.
```